routes/routes.go

```diff
@@ -27,8 +26,17 @@ type Clients struct {
 
 // Setup registers routes for the service
 func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, c Clients, cacheList cache.List) {
-	log.Info(ctx, "adding routes")
 	r.StrictSlash(true).Path("/health").HandlerFunc(c.HealthCheckHandler)
-	r.StrictSlash(true).Path("/search").Methods("GET").HandlerFunc(handlers.Read(cfg, c.Zebedee, c.Renderer, c.Search, cacheList))
+	r.StrictSlash(true).Path("/search").Methods("GET").HandlerFunc(handlers.Read(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "search"))
+
+	r.StrictSlash(true).Path("/alladhocs").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "all-adhocs"))
+	r.StrictSlash(true).Path("/datalist").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-datalist"))
+	r.StrictSlash(true).Path("/publishedrequests").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "published-requests"))
+	r.StrictSlash(true).Path("/staticlist").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-list"))
+	r.StrictSlash(true).Path("/topicspecificmethodology").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-methodology"))
+	r.StrictSlash(true).Path("/timeseriestool").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "time-series-tool"))
+	r.StrictSlash(true).Path("/publications").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-publications"))
+	r.StrictSlash(true).Path("/allmethodologies").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "all-methodologies"))
+
```

Comment:
These paths should sit behind a feature toggle

Reply:
Not sure as to what that toggle should be as the old implementation didn't have these routes as toggleable

Reply:
The old implementation is live in production so doesn't sit behind a feature toggle. If these changes go to prod they would become the live endpoints. It could be something like `ENABLE_NEW_DATA_AGGREGATION` or something more creative